packages/ketcher-core/src/application/render/restruct/resgroup.js

```diff
@@ -27,13 +27,19 @@ import { tfx } from 'utilities'
 class ReSGroup extends ReObject {
   constructor(sgroup) {
     super('sgroup')
+    /** @type {SGroup} */
```

Comment:
I propose to use TS types instead.

Reply:
this fils is Js, is your proposal to refactor this file fully? If yes - it should be part of refactoring PR as it will affect a lot of lines.